CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ A more detailed list of changes is available in the corresponding milestones for
 ## 0.8.0 (2020-Jul-??)
 ### Note-worthy code changes
   - remove PriorityLevel class as it makes classifying checks by priority more complicated then necessary! (issue #2981)
-
+  - Added **[com.google.fonts/check/varfont_duplicate_instance_names]**: Avoid duplicate instance names in variable fonts (issue #2986)
+  - ...
```

Comment:
sorry for being picky!

We delete the `...` line once we have some content and we also have a separate section called `### New checks` in the changelog of all releases. Please take a look at past releases and follow the same style